homeassistant/components/esphome/assist_satellite.py

```diff
@@ -344,6 +368,11 @@ def handle_timer_event(
             timer_info.is_active,
         )
 
+    async def handle_announce_finished(self, media_id: str) -> None:
+        """Handle when announcement is finished on satellite."""
+        _LOGGER.debug("Announcement finished: %s", media_id)
+        self._announcement_finished.set()
```

Comment:
Let's not keep it around:
```suggestion
        self._announcement_finished.set()
        self._announcement_finished = None
```